dicomImport/matRad_calcWaterEqD.m

```diff
@@ -32,61 +32,18 @@
 %
 % %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+% set conversion to HU to EqD if conversionMode is not provided
+% if (~exist('conversionMode', 'var'))
+%     conversionMode = HU2EqD;
+% end
+
```

Comment:
What is this? This variable is never used and also never passed to the function -->the if statement is always true.